scripts/tools/zap/run_zaptool.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+CHIP_ROOT=$(readlink -f ${BASH_SOURCE%/scripts/tools/zap/run_zaptool.sh})
+ZAP_ROOT="$CHIP_ROOT/third_party/zap/repo"
+
+(
+	cd $CHIP_ROOT
+	git submodule update --init third_party/zap/repo
```

Comment:
```suggestion
	cd $CHIP_ROOT \
	  && git submodule update --init third_party/zap/repo \
	  || exit 1
```
not to continue if any intermediate step fails. Same for other instructions below.